clouddq/utils.py

```diff
@@ -254,3 +269,13 @@ def update_dict(dict1: dict, dict2: dict) -> dict:
             output_dict[key].append(dict2[key])
 
     return output_dict
+
+
+def transform_json_str_to_dict(value: dict) -> dict:
+
+    return dict(
+        {
+            "id": value["id"],
+            "include_reference_columns": eval(value["include_reference_columns"]),
```

Comment:
When we fetch the reference columns from sqlite db, it is returned as a json string 
`{'id': 'CONTACT_DETAILS_REFERENCE_COLUMNS', 'include_reference_columns': '["row_id", "contact_type", "value"]'}`. This method will transform it to dictionary. We can move this method from utils.py to DqReferenceColumns class. The alternate approach in case `eval` has some security issues is the explicit conversion to list from string represenation of list using `include_reference_columns.strip('[]').replace('"', "").split(', ')`

Reply:
Thanks for the explanation Amandeep. Could you please move this inside of the DqReferenceColumns class? Is it possible to use json.loads() to convert the json string to a dict?